src/__tests__/organization.tests.ts

```diff
@@ -0,0 +1,301 @@
+        });
+    
+        it('When updating new organization with invalid polygon, Expect to return HTTP Status 400', async () => {
+          //Arrange
+          let oraganizationApi = new OrganizationApi(configurationWithAuthHeader);
+          //Act
+          let payload = seederData?.organization!;
+          payload.polygon = TdeiObjectFaker.getInvalidPolygon();
+    
+          const oraganizationResponse = oraganizationApi.updateOrganization(payload);
+    
+          //Assert
+          await expect(oraganizationResponse).rejects.toMatchObject({ response: { status: 400 } });
+         });
+        });
+      });
+
+      describe('Get Organizations', () => {
+        describe('Auth', () => {
+          it("When no auth token provided, Expect to return HTTP status 401", async () => {
+            //Arrange
+            let oraganizationApi = new OrganizationApi(configurationWithoutAuthHeader);
+            //Act
+            const organizationResponse = oraganizationApi.getOrganization();
+          
+            //Assert
+            expect(organizationResponse).rejects.toMatchObject({response: { status: 401 } })
+          });
+        });
+
+        describe('Functional', () => {
+          it('When searched without filters, Expect to return list of organizations of type OrganizationList', async () => {
+            //Arrange
+            let oraganizationApi = new OrganizationApi(configurationWithAuthHeader);
+
+            //Act
+            const oraganizationResponse = await oraganizationApi.getOrganization();
+
+            //Assert
+            expect(oraganizationResponse.status).toBe(200);
+            expect(oraganizationResponse.data).toBeInstanceOf(Array);
+            });
+
+            it('When searched with tdei_org_id filter, Expect to return list of organizations matching filter', async () => {
+            //Arrange
+            let oraganizationApi = new OrganizationApi(configurationWithAuthHeader);
+
+            //Act
+            const oraganizationResponse = await oraganizationApi.getOrganization(seederData?.organizationId, undefined, undefined, undefined, undefined);
+
+            const data = oraganizationResponse.data;
+
+            //Assert
+              expect(oraganizationResponse.status).toBe(200);
+              expect(Array.isArray(data)).toBe(true);
+              expect(data).toBeInstanceOf(Array);
+              expect(data[0].tdei_org_id).toEqual(seederData?.organizationId);
+            });
+
+            it('When searched with organization name filter, Expect to return list of organizations matching fiter', async () => {
+            //Arrange
+            let oraganizationApi = new OrganizationApi(configurationWithAuthHeader);
+
+            //Act
+            const oraganizationResponse = await oraganizationApi.getOrganization(undefined, seederData?.organization?.org_name, undefined, undefined, undefined);
```

Comment:
parameters after the org_name are not needed.

Reply:
fixed